cmd/broscoli/main_test.go

```diff
@@ -0,0 +1,136 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"github.com/hypebeast/go-osc/osc"
+	"gopkg.in/yaml.v2"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"strings"
+	"testing"
+	"time"
+)
+
+const (
+	SubCmdFlags = "SUB_CMD_FLAGS"
+)
+
+func TestAPI(t *testing.T) {
+	_ = os.Remove("/tmp/foo")
```

Comment:
Why the remove?

Reply:
The test is testing that `touch /tmp/foo` is executed by checking the presence of a file after the server receives a message.